api/src/main/java/org/pmiops/workbench/workspaces/resources/WorkspaceFields.java

```diff
@@ -13,7 +13,6 @@
   private BillingStatus workspaceBillingStatus;
   private String cdrVersionId;
   private String accessTierShortName;
-  private String permission;
```

Comment:
this field isn't derived from `DbWorkspace`, and it became awkward to include it here